dali/imgcodec/util/output_shape.h

```diff
@@ -44,15 +44,24 @@ void OutputShape(OutShape &&out_shape,
     if (rotate) {
       in_d = 1 - d;
     }
+
     int extent = info.shape[in_d];
-    if (d < roi.end.size())
-      extent = roi.end[d];
-    if (d < roi.begin.size())
-      extent -= roi.begin[d];
+    if (roi) {
```

Comment:
This check seems redundant...

Reply:
I will double check it, I think I had some error without it. But yes, seems redundant.

Reply:
Okay, I think I know why I did this, but that was wrong.